packages/ui/src/services/color-config/color-css-helpers-updater.ts

```diff
@@ -0,0 +1,54 @@
+import { watch } from 'vue'
+import { useGlobalConfig } from '../global-config/global-config'
+import { isServer } from '../../utils/ssr-utils'
+import { addOrUpdateStyleElement } from '../dom-functions'
+import type { ColorConfig } from './color-config'
+
+export type HelperConfig = {
+  stylePrefix: string;
+  styleProperty: string;
+}
+
+export const helperConfigValues: HelperConfig[] = [
+  {
+    stylePrefix: 'bg',
+    styleProperty: 'background-color',
+  },
+  {
+    stylePrefix: 'text',
+    styleProperty: 'color',
+  },
+]
+
+export const renderCSSHelpers = (helpers: ColorConfig) => {
+  const helperNames = Object.keys(helpers)
+  let resultHelperClasses = ''
+
+  helperNames.forEach((key) => {
+    helperConfigValues.forEach((helper: ColorConfig) => {
+      resultHelperClasses += `.${helper.stylePrefix}--${key} {${helper.styleProperty}: ${helpers[key]};}`
+    })
+  })
+
+  return resultHelperClasses
+}
+
+const ColorHelpersPlugin = {
+  install () {
+    if (isServer()) {
+      return
+    }
+
+    const { globalConfig } = useGlobalConfig()
+
+    watch(() => globalConfig.value.classHelpers, (newValue) => {
+      if (!newValue) {
+        return
+      }
+
+      addOrUpdateStyleElement('css-helpers', () => renderCSSHelpers(newValue))
```

Comment:
```suggestion
      addOrUpdateStyleElement('va-css-helpers', () => renderCSSHelpers(newValue))
```

Reply:
Considering that starting from Vue3 user can have many vue apps on same page, maybe we need to add vue app uid as suffix. But, let's think about it later in a different task.